Client/Ecosia/IncentivizedSearchChecker/IncentivisedSearchChecker.swift

```diff
@@ -0,0 +1,17 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Core
+
+struct IncentivisedSearchChecker {
+    private static let checker = IncentivizedSearchCheckerWithFallbackComposite(
+        primary: UnleashIncentivizedSearchChecker(),
+        fallback: LocalIncentivizedSearchChecker()
+    )
+    
+    static var isSearchIncentiveRestricted: Bool {
```

Comment:
also this could be simply `isRestricted` as the class name already tells that it's about search incentives. Thus consuming calls would be a bit shorter: `IncentivisedSearchChecker.isRestricted`

I am keen to shorten this API even further, that so it could be used in ternary expressions more readably.